piyushgarg-dev/review-app
# fix: Fixed the background color of Form Preview to grey shade on dark mode

## What does this PR do?

This PR basically turns the background color of the Form Preview to grey shade when the dark mode is turned on by the user and it feels much more comfortable using the application.

Fixes #130 

BEFORE

https://github.com/piyushgarg-dev/review-app/assets/131521505/e45d2c71-68eb-4aec-8012-9466542a1e9d

AFTER

https://github.com/piyushgarg-dev/review-app/assets/131521505/e88a31c5-dbf4-4785-be0e-8a2f35be0621

## Type of change

## How should this be tested?

1. Create a project
2. Create a form
3. Click on edit option.
4. Observe the background color where the form is previewed.

If you already created a form in a project, then click on this click to go directly to the form page http://app.localhost:3000/dashboard/subdomain/forms and click on edit icon.

## Mandatory Tasks
